pathology/hovernet/hovernet_training_torch.ipynb

```diff
@@ -23,7 +23,7 @@
     "\n",
     "The CoNSeP dataset comes from https://warwick.ac.uk/fac/cross_fac/tia/data/hovernet/.\n",
     "\n",
-    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Project-MONAI/tutorials/blob/main/pathology/hovernet/training_torch.ipynb)"
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/Project-MONAI/tutorials/blob/main/pathology/hovernet/hovernet_training_torch.ipynb)"
```

Comment:
Maybe changing the name of the notebook to `hovernet_torch` as it already includes training, validation and also sliding inference. what do you think?

Reply:
Yes, I think so, have renamed the file name! Thanks!